js/exercise-03.js

```diff
@@ -0,0 +1,17 @@
+/* 
+    1. Define function 'getTypeOf', which receives a value as paramenter and returns the type of this value
+        Use typeof to ckech the types for each of the main 5 primitive types (string, number, boolean, null, undefined)
+    2. Log the result of any function call.
+*/
+
+function getTypeOf(value) {
```

Comment:
There is an easiest way to define this function.

What does typeof return? Did you read the [documentation](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/typeof)? 

it's not necessary to map types... 

FIX: find the right way to implement it. 🚀